thousandeyes/data_source_thousandeyes_account_group.go

```diff
@@ -40,7 +40,7 @@ func dataSourceThousandeyesAccountGroupRead(d *schema.ResourceData, meta interfa
 	var found *thousandeyes.SharedWithAccount
 
 	for _, account := range *accounts {
-		if account.AccountGroupName == searchName {
+		if *account.AccountGroupName == searchName {
```

Comment:
As a suggestion, I believe that a getter that returns the value or empty if null makes this easier to read. Pointers can be error prone IMO.

Reply:
That'd be a good improvement for `go-thousandeyes`. I'll take note of it.